ILSpy.Core/Commands/OpenFromNuGetCommand.cs

```diff
@@ -30,6 +30,7 @@ sealed class OpenFromNuGetCommand : SimpleCommand
 		public override async void Execute(object parameter)
 		{
             OpenFileDialog dlg = new OpenFileDialog();
+			dlg.Title = "Open file";
```

Comment:
Looking at the coding style, this file uses a mix of spaces and tabs, but the surrounding context of this line is indented with spaces and so should this. Might be worth enabling "render whitespace" (or equivalent) in your editor to see what is going on and adjust accordingly.